src/lib/lightning/lnd/mappers.ts

```diff
@@ -25,4 +26,5 @@ export const mapPendingChannel = (status: LightningNodeChannel['status']) => (
   localBalance: chan.localBalance,
   remoteBalance: chan.remoteBalance,
   status,
+  isPrivate: false,
```

Comment:
Since `private` field doesn't exist on `PendingChannel` type so I'm setting it to `false` here. I guess I could also make the field nullable instead and assign it to `null` here?

Reply:
Using `false` is fine with me here.

Reply:
Cool.